app/router.ts

```diff
@@ -142,6 +142,7 @@ Router.map(function() {
         this.mount('analytics-page', { as: 'analytics' });
         this.route('forks');
         this.route('registrations');
+        this.route('drafts', { path: '/drafts/:draft_id' });
```

Comment:
also camelcase

Reply:
This will require disabling `ember/routes-segments-snake-case`, which we should do globally in `.eslintrc.js`.